app/views/admin/claims/_claim.html.erb

```diff
@@ -0,0 +1,5 @@
+<tr class="govuk-table__row">
+  <th scope="row" class="govuk-table__header"><%= claim.reference %></th>
+  <td class="govuk-table__cell"><%= claim.first_name %> <%= claim.surname %></td>
```

Comment:
We have logic to create this already for the emails. Can we avoid duplication?

Reply:
Ah, I thought we'd got rid of that